canonical/ubuntu.com
# Remove duplicate ID's

## Done

This work is part of the accessibility audit, it is removing duplicate ID from the same page. This work *does not* cover the kubernetes docs bubble, and will be tackled in a [this seperate issue](https://warthogs.atlassian.net/browse/WD-1737).
This file contains a list of al the duplicate ID's, the ones highlighted in green have been completed in this PR [Duplicate.id.-.the.same.ID.is.used.on.more.than.one.element.ods](https://github.com/canonical/ubuntu.com/files/10545855/Duplicate.id.-.the.same.ID.is.used.on.more.than.one.element.ods)


## QA

- Go through each item on the list and find the affected pages, go to the page and inspect the elements. Use the `find` tool to search for the listed ID, make sure there are no duplicates.
**FYI, the spreadsheet is a bit of mess, if you need help interpreting it, please reach out**
- If a change has been applied to a page with functionality, check it is still working as expected, ex. templates/certified/search-results.html
- If a change has occurred involving a marketo form, check the correct for is still associated to the page

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1561
